.github/workflows/daily.yml

```diff
@@ -123,7 +129,7 @@ jobs:
           type: 'stream'
           topic: 'lean4checker'
           content: |
-            ✅ lean4checker [succeeded](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}) on ${{ github.sha }} (branch: ${{ env.BRANCH_REF }})
+            ✅ lean4checker [succeeded](${{ steps.urls.outputs.lean4checker_url }}) on ${{ github.sha }} (branch: ${{ env.BRANCH_REF }})
```

Comment:
We could also potentially reword this to mention the other thread to make it even more clear that this failure is specific to `lean4checker`? But I thought that might be too wordy so I just changed the URL.

Reply:
Sounds good.